webapp/channels/src/components/admin_console/permission_schemes_settings/permissions_scheme_summary/permissions_scheme_summary.tsx

```diff
@@ -148,17 +146,13 @@ export default class PermissionsSchemeSummary extends React.PureComponent<Props
         let extraTeams = null;
         if (teams.length > MAX_TEAMS_PER_SCHEME_SUMMARY) {
             extraTeams = (
-                <OverlayTrigger
-                    delayShow={Constants.OVERLAY_TIME_DELAY}
-                    placement='bottom'
-                    overlay={
-                        <Tooltip
-                            className='team-scheme-extra-teams-overlay'
-                            id={scheme.id + '-extra-teams-overlay'}
-                        >
+                <WithTooltip
+                    id={scheme.id + '-extra-teams-overlay'}
+                    title={
```

Comment:
```
title={this.props?.teams?.slice(MAX_TEAMS_PER_SCHEME_SUMMARY).map((team) => team.display_name).join(', ') ?? ''}
```

I observed that we were just not adding any space between team names, so with the above it will add comma between each so lets do that instead and we dont need to have classname then.